src/main/java/org/embulk/output/bigquery_java/BigqueryJavaOutputPlugin.java

```diff
@@ -111,7 +111,9 @@ public ConfigDiff transaction(ConfigSource config,
         }
 
         if (task.getTempTable().isPresent()) {
-            if (task.getMode().equals("append")) {
+            if (task.getMode().equals("merge")) {
+                client.merge(task.getTempTable().get(), task.getTable(), task.getMergeKeys().orElse(Collections.emptyList()), task.getMergeRule().orElse(Collections.emptyList()));
```

Comment:
Table copy job can only use `WRITE_TRUNCATE` or `WRITE_APPEND`, so use query job to merge instead.
https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#jobconfigurationtablecopy
https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#jobconfigurationquery